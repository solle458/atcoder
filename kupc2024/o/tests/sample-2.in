10
2 4
3 8
7 9
6 3
2 1
9 5
9 6
3 10
9 2
19359725 62617217 96402215 25620070 46288197 40930485 37441368 32164537 69968419 79525817
