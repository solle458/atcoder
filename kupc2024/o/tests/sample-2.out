1525009090 2231445426
